src/Core/Components/Tabs/FluentTab.razor.cs

```diff
@@ -28,6 +28,18 @@ public partial class FluentTab : FluentComponentBase
     [Parameter]
     public bool Disabled { get; set; } = false;
 
+    /// <summary>
+    /// Gets or sets the visability of a tab
+    /// </summary>
+    [Parameter]
+    public bool Visible { get; set; } = true;
+
+    /// <summary>
+    /// Callback to invoke when visibility changes
+    /// </summary>
+    [Parameter]
+    public EventCallback<bool> VisibleChanged { get; set; }
```

Comment:
Why to have a `VisibilityChanged`, if this event is never called by the component?

Reply:
I was proactive and thought we would probably need it somewhere? 😂

Reply:
Mmmm ? No way 😀